doc/nrf/links.txt

```diff
@@ -95,6 +93,18 @@
 .. _`Project CHIP Protocol Overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/README.md
 .. _`nRF Connect platform overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/docs/guides/nrfconnect_platform_overview.md
 
+.. _`Azure support for Asset Tracker v2`: https://github.com/nrfconnect/sdk-nrf/pull/3950
```

Comment:
Are you sure you want to link to a pull request?

Reply:
As pointed out by Pekka and Uma, this is addressed in https://github.com/nrfconnect/sdk-nrf/pull/3950